### ISSUE TITLE
Wrong text encoding when open some wmf image contain Chinese

### ISSUE THREAD

**Issue by kenifanying** [ONLYOFFICE/DesktopEditors#1031]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Windows 10 / 8.1 / 7

### Version information

7.2.0.204

### Expected Behavior

Image correctly displayed

### Actual Behavior

Wrong text encoding when open some wmf image

onlyoffice  vs ms office

<img width="860" alt="encoding-test" src="https://user-images.githubusercontent.com/4479181/192617390-085efcc0-ee70-437e-8e15-e79496134ac5.png">

### Reproduction Steps

open the test file

[encoding-test.docx](https://github.com/ONLYOFFICE/DesktopEditors/files/9659032/encoding-test.docx)

**Comment by TatianaKirillova** [comment]
Hey, [kenifanying](https://github.com/kenifanying)

Thanks for your request!

The picture is displayed correctly. I attach the display of the picture itself in the Paint editor, as well as in the Onlyoffice editor and MS Word 2019.
![image1 wmf - Paint 2023-08-02 12 56 35](https://github.com/ONLYOFFICE/DesktopEditors/assets/72191601/c727061b-98b4-456f-b96b-007ad175e18a)

**Comment by kenifanying** [comment]
> The picture is displayed correctly. I attach the display of the picture itself in the Paint editor, as well as in the Onlyoffice editor and MS Word 2019.

Maybe related to encoding environment, I test it under Windows 7/10/11 Simplified Chinese environment which codepage is  cp936.  As you can see, the original snapshot on the right can display the correct character which is `9 x 270 x N个字节`.


![image](https://github.com/ONLYOFFICE/DesktopEditors/assets/4479181/c978f217-d515-49e8-93df-93e34f1b7c92)

**Comment by TatianaKirillova** [comment]
@kenifanying, Do you open MS Word editor and Onlyoffice editor on the same computer?

**Comment by kenifanying** [comment]
> @kenifanying, Do you open MS Word editor and ONLYOFFICE editor on the same computer?

Yes,  see my original attachment. I take the snapshot side by side on the same computer. (left: onlyoffice, right: ms office)

**Comment by TatianaKirillova** [comment]
Thank, @kenifanying !
I can confirm that the problem exists. I created ticket 63565 in our internal  tracker with your  issue.
